pkg/webui/console/components/rights-group/index.js

```diff
@@ -140,15 +146,21 @@ class RightsGroup extends React.Component {
         className={style.rightLabel}
         key={right}
         name={right}
-        label={{ id: `enum:${right}` }}
         disabled={outOfOwnScopeRights.includes(right)}
+        label={
+          <React.Fragment>
+            <Message content={{ id: `enum:${right}` }} />{' '}
+            {outOfOwnScopeRights.includes(right) && (
+              <Message className={style.revoke} content={m.revocableOnly} />
```

Comment:
I would prefer making this a custom checkbox component

Reply:
I think to pass the additional content as children will simplify this enough.